src/components/pages/HeadsUpDisplay/CurrentActivity.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect } from 'react';
+
+export const CountDownTimer = () => {
+  // This function will calculate difference between current and target times
+  const calculateTimeRemaining = () => {
+    let currTime = new Date();
+    // let currTime = new Date("2021-08-08T23:59:59");
+    // let currTime = new Date("2021-08-09T00:00:00");
+    // console.log("currTime", currTime);
+
+    // If the day of week is Saturday, countdown will be 48 hours, otherwise, 24 hours
+    let endTime = new Date().setHours(24, 0, 0, 0);
+    // console.log("endTime", convertTimeFormat(endTime));
+    // currTime.getDay() === 6
+    //   ? new Date().setHours(48, 0, 0, 0)
+    //   : new Date().setHours(24, 0, 0, 0);
+
+    // let endTime = new Date("2021-08-09T00:00:00");
+
+    let timeDiff = endTime - currTime;
+
+    // console.log("timeDiff", convertTimeFormat(timeDiff));
```

Comment:
Any comments of code can be removed if they are not explaining how to it works

Reply:
This code is from a component that I have since refactored, I'm not sure how these got pushed up, I only added the countdown timer to this feature. The final current activity component is complete and free of console.logs.